proto/pulumi/resource.proto

```diff
@@ -250,5 +254,13 @@ message RegisterProviderRequest {
 
 message RegisterProviderResponse {
     // The UUID package reference for this registered provider package.
-    string ref = 1; 
-}
\ No newline at end of file
+    string ref = 1;
+}
+
+message CreateNewContextRequest {
+    repeated string providers = 1; // the provider refs to register as the default providers.
+}
+
+message CreateNewContextResponse {
+    string monitorAddr = 1;
```

Comment:
```suggestion
    string monitor_target = 1;
```
proto files ought to be snake_case for field names (we we're inconsistent about this before). Also we're trying to call these grpc address "target", that matches the ones for the loader and mapper.

Reply:
Done :heavy_check_mark: